lib/validators/actions.js

```diff
@@ -120,6 +120,114 @@ export function runMetaValidators(uri, errorValidators, warningValidators) {
     .catch(log.error);
 }
 
+/**
+ * Check if the validationRule has the correct properties to run validation with
+ * @param {object} validators
+ * @param {object} validationRule
+ * @return {boolean}
+ */
+function validKilnjsValidator(validators, validationRule) {
+  return  validationRule.validator &&
+          validators[validationRule.validator] &&
+          validators[validationRule.validator].type &&
+          validators[validationRule.validator].kilnjsValidate;
+}
+
+/**
+ * Run the kilnjsValidators
+ * @param {object} schemas
+ * @param {object} components
+ * @return {array}
+ */
+export function runKilnjsValidators(schemas = {}, components = {}) {
+  const { validators } = window.kiln;
+  let errors = [];
+
+  Object.keys(components).forEach((uri) => {
+    const componentName = getComponentName(uri);
+
+    if (schemas[componentName]) {
+      schemas[componentName].validation.forEach((validationRule) => {
+        if (validKilnjsValidator(validators, validationRule)) {
+          const valueField = getKilnJsFieldAndValue(validationRule, components[uri]),
+            validationError = validators[validationRule.validator].kilnjsValidate(valueField.value);
+
+          if (validationError) {
+            const errorItem = {
+              uri,
+              field: valueField.fieldName,
+              location: componentName,
+              preview: validationError.preview,
+              path: getPathFromField(uri, valueField.fieldName)
+            };
+
+            errors = getKilnJsError(validationError, errorItem, errors);
+          }
+        }
+      });
+    }
+  });
+
+  return errors;
+}
+
+/**
+ * Get the schemas that have a validation property
+ * @param {object} schemas
+ * @returns {object}
+ */
+export function getSchemasWithValidationRules(schemas = {}) {
+  const validationSchemas = {};
+
+  Object.keys(schemas).forEach((schemaName) => schemas[schemaName].validation ? validationSchemas[schemaName] = _.cloneDeep(schemas[schemaName]) : null );
```

Comment:
Can you please break this line into multiple lines? Just to make it more readable
```suggestion
  Object.keys(schemas).forEach((schemaName) => (
    schemas[schemaName].validation
      ? validationSchemas[schemaName] = _.cloneDeep(schemas[schemaName])
      : null
    )
  );
```